compiler/commands.nim

```diff
@@ -234,15 +234,20 @@ const
   errGuiConsoleOrLibExpectedButXFound = "'gui', 'console' or 'lib' expected, but '$1' found"
   errInvalidExceptionSystem = "'goto', 'setjump', 'cpp' or 'quirky' expected, but '$1' found"
 
+template deprecatedSwitch(switch: string) =
+  warningDeprecated(conf, info, "'$#' is a deprecated noop" % switch)
```

Comment:
```suggestion
  warningDeprecated(conf, info, "'$#' is deprecated" % switch)
```
Because that template by itself can not confirm it is *always* a noop.

Reply:
rephrased and renamed to `warningOptionNoop` to make it clear to only use it for noop